test/powershell/Host/TabCompletion/BugFix.Tests.ps1

```diff
@@ -21,4 +21,11 @@ Describe "Tab completion bug fix" -Tags "CI" {
         $result.CompletionMatches[0].CompletionText | Should Be "-and"
         $result.CompletionMatches[1].CompletionText | Should Be "-as"
     }
+    It "Issue#2295 - '[pscu<tab>' should expand to [pscustomobject]" {
+        $result = TabExpansion2 -inputScript "[pscu" -cursorColumn "[pscu".Length
+        $result | Should Not BeNullOrEmpty
+        $result.CompletionMatches.Count | Should Be 2
+        $result.CompletionMatches[0].CompletionText | Should Be "pscustomobject"
+        $result.CompletionMatches[1].CompletionText | Should Be "psobject"
```

Comment:
As per the current behavior, pscu expands to System.Management.Automation.PSCustomObject.  After this change, it will now expand to pscustomobject and psobject. both of these accelerators refer to System.Management.Automation.PSObject. I am afraid I don't understand your concern here.